strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/config/util/UpdatableMultipartConfigElement.java

```diff
@@ -0,0 +1,29 @@
+package org.carlspring.strongbox.config.util;
+
+import javax.servlet.MultipartConfigElement;
+
+public class UpdatableMultipartConfigElement extends MultipartConfigElement
+{
+
+    private volatile long maxFileSize = -1;
+
+    public UpdatableMultipartConfigElement(String location,
+                                           long maxFileSize,
+                                           long maxRequestSize,
+                                           int fileSizeThreshold)
```

Comment:
Use `final` modifier when possible.
```suggestion
    public UpdatableMultipartConfigElement(final String location,
                                           final long maxFileSize,
                                           final long maxRequestSize,
                                           final int fileSizeThreshold)
```